spec/spec_helper.rb

```diff
@@ -1,5 +1,9 @@
+require 'simplecov'
+SimpleCov.start
```

Comment:
It adds noise. I don't need noise. Look at the output:

![rspec](https://cloud.githubusercontent.com/assets/297060/7779891/a8409abc-00a6-11e5-81f0-20698272ab4a.png)

I want to see something like
```
...............

Finished in X
N examples, 0 failures
```

That's it. The coverage thingy is not a useful metric to me.